### PR TITLE
Drop obsolete wayland compositors

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

- [x] pkgs/applications/window-managers/orbment/bemenu.nix
- This is now replaced by https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/bemenu/default.nix
- [x] pkgs/applications/window-managers/orbment/default.nix
- Not updated since 2016

- [x] pkgs/applications/window-managers/velox/default.nix
- Not used by anyone but its author, has no active development, all of its components are not compatible with other compositors
- [x] pkgs/applications/window-managers/velox/dmenu.nix
- [x] pkgs/applications/window-managers/velox/st.nix
- [x] pkgs/applications/window-managers/velox/swc.nix
- [x] pkgs/applications/window-managers/velox/wld.nix

- [x] pkgs/development/libraries/wlc/default.nix
- Move to `pkgs/applications/window-managers/way-cooler/wlc.nix` nothing but `way-cooler` currently uses it. Once there will be release of way-cooler using `wlroots`, we will drop wlc.nix

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by gnidorah** [comment]
@primeos What do you think?

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -18401,10 +18398,7 @@ in
     pulseSupport = config.pulseaudio or false;
   };
 
-  velox = callPackage ../applications/window-managers/velox {
-    stConf = config.st.conf or null;
-    stPatches = config.st.patches or null;
-  };
+  way-cooler = callPackage ../applications/window-managers/way-cooler {};
```
Comment (primeos) [inline pkgs/top-level/all-packages.nix]: Any reason for moving `way-cooler` here?

Normally the attributes within categories should be sorted alphabetically, but especially this part of the file is pretty messed up anyway...

**Review by primeos — APPROVED** [review]
LGTM, thanks :)

**Comment by primeos** [comment]
cc @tg-x

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (gnidorah) [reply pkgs/top-level/all-packages.nix]: @primeos No real reason, I've reverted that change

**Comment by gnidorah** [comment]
@primeos merge please?

**Comment by primeos** [comment]
@gnidorah sorry, forgot to merge it :o Thanks for the reminder and the PR :)

**Comment by gnidorah** [comment]
@primeos Thank you!

**Pull request merged**